src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const resultObj = {};
+  const createdArray = sourceString.split(';');
+
+  createdArray.map((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      resultObj[key.trim()] = value.trim();
+    }
+  });
```

Comment:
The map() method should return some value. It doesn't fit here. Rewrite it by using forEach, please